### PR TITLE
Feature/auth improvements 245

### LINKED ISSUE #245: Create account and Sign in improvements
After the usability test, we must change some changes on the Registration and Sign in flow to improve the user experience:

1. The pdf file must download in the background without showing to the user. In iOS the pdf file must be open in another tab but do not be shown.

**This item was removed from the task:**
> 2. In the dashboard, we must show the tip on the header. When the user clicks, he must be redirected to another page that shows complete information (figma). After that, this tip should never appear again.
> 
> Figma screen (a tip on the header):
> https://www.figma.com/file/47BEsGEMSWfhTm3zC0ugyA/Main-flow?node-id=556%3A13
> 
> Figma screen (tip details):
> https://www.figma.com/file/47BEsGEMSWfhTm3zC0ugyA/Main-flow?node-id=556%3A381

3. Change the text on Basic informations step:

- Button text: "Continue"

Figma screen:
https://www.figma.com/file/47BEsGEMSWfhTm3zC0ugyA/Main-flow?node-id=3%3A2206

4. Change the text on 12 words step:

- "Copy and paste the 12 words below in a secure place. You have only this chance to do it; if you lost these words, you can't login into your account never more."

- Button text: "Create my account"

Figma screen:
https://www.figma.com/file/47BEsGEMSWfhTm3zC0ugyA/Main-flow?node-id=3%3A2224

5. Change the text on Login (12 words form):

- "Enter your 12 words, that you've copied and pasted on registration."

- Add a placeholder on 12 words input: "Type your 12 words here"

Figma screen:
https://www.figma.com/file/47BEsGEMSWfhTm3zC0ugyA/Main-flow?node-id=3%3A2590

### PR DESCRIPTION
**Pull request by amiskov**
## What issue does this PR close
Closes #245 

## Changes Proposed (a list of new changes introduced by this PR)
- Update texts for Register and Login screens.
- Add small UI improvements here and there: autofocus for passphrase/PIN inputs, loading indicator for submitting login form, translations for some texts.
- ~~Improve PDF flow.~~ It looks like we can't achieve downloading PDF in the background for the iOS Safari. See this [conversation](https://github.com/cambiatus/frontend/issues/245#issuecomment-665228430) for the details.

## How to test (a list of instructions on how to test this PR)
- Open Register/Login screens, see texts changed.
- ~~PDF should download differently.~~

### CODE REVIEW

File: public/translations/pt-BR.json [hunk 1]
```diff
@@ -194,8 +197,8 @@
       "twelve_words": "Suas 12 palavras",
       "private_key": "Sua chave privada",
       "recommendation": "Recomendamos que você salve essas informações clicando no botão abaixo. Se você perder, não poderá fazer login novamente na sua conta.",
-      "download": "Salvar chave privada e 12 palavras",
-      "instructions": "Você salvará a chave privada e as 12 palavras em seu computador no formato PDF."
+      "download": "Criar a minha conta",
+      "instructions": "Copie e cole as 12 palavras abaixo em um local seguro. Você tem apenas essa chance de fazê-lo; se você perdeu essas palavras, nunca mais poderá acessar sua conta."
```
Comment (lucca65) [inline public/translations/pt-BR.json:201]: maybe a more "common" way would be 

> Copie e cole as 12 palavras abaixo em um local seguro. Essa é a sua única chance de fazê-lo; em caso de perda das palavras, você não poderá mais recuperar ou acessar sua conta 

Any thoughts @heltonlr @RodrigoLeiteF

File: src/elm/Auth.elm [hunk 2]
```diff
@@ -649,6 +664,10 @@ update msg shared model =
                         , status = Options LoginStepPIN
                     }
                         |> UR.init
+                        |> UR.addCmd
+                            (Dom.focus "pinInput"
+                                |> Task.attempt (\_ -> Ignored)
```
Comment (lucca65) [inline src/elm/Auth.elm:669]: 😌

File: src/elm/Session/Guest.elm [hunk 3]
```diff
@@ -126,7 +126,7 @@ view thisMsg page ({ shared } as model) content =
                   else
                     text ""
                 , div
-                    [ class "min-h-stretch flex flex-col"
+                    [ class "min-h-screen flex flex-col"
```
Comment (lucca65) [inline src/elm/Session/Guest.elm:129]: oh I see @amiskov 

But i wonder why i couldn't reproduce this behavior from my machine, only after resizing the browser (before this commit)

File: public/translations/pt-BR.json (hunk 1 above)
Comment (heltonlr) [reply public/translations/pt-BR.json:201]: Works to me

File: src/elm/Session/Guest.elm (hunk 3 above)
Comment (amiskov) [reply src/elm/Session/Guest.elm:129]: I tried to understand this but with no success... 🤷🏻‍♂️ Even `min-h-stretch` is not working anymore, but there were reasons for me to use it earlier! Maybe we changed something in the markup in previous commits in the past couple of months, but I didn't find anything suspicious...

**Pull request merged**